ethereum/remix-project
# fix vm fork

rewrite of the CustomEthersStateManager.
the previous implementation didn't handle well the checkpoint/revert.

in order to test it:

 - compile
```
contract Campaign {
    uint public cake;
    function contribute() public payable {
       cake++;
    }
}
```

- deploy and call contribute
- access "cake", this should print "1"
- call contribute again
- access "cake", this should print "2"
